group/assignment6/RearrangeCars.cpp

```diff
@@ -140,33 +120,38 @@ void RearrangeCars::generateSequenceOfMovesWithConstraints(const vector<int> &en
 	}
 }
 
-void RearrangeCars::bruteForceAllSequence(vector<int>& currentStates, const vector<int>& endStates, const vector<set<int>> &constraints, vector<int>& carPosition, int emptySlotId, vector<Move>& sequenceOfMoves, vector<vector<Move>>& sequencesOfMoves, set<vector<int>>& usedPositions) {
+void RearrangeCars::bruteForceAllSequence(vector<int>& currentStates, const vector<int>& endStates, vector<int>& carPosition, int emptySlotId, vector<Move>& sequenceOfMoves, vector<vector<Move>>& sequencesOfMoves, set<vector<int>>& usedPositions, const vector<set<int>> *constraints) {

```

Comment:
I don't like the function name bruteForceAllSequence :) call it generateMoves or findAllSequencesOfMoves

Reply:
Agree.

Reply:
Me too